CRM/Activity/BAO/Activity.php

```diff
@@ -2511,7 +2509,7 @@ public static function getContactActivitySelector(&$params) {
         }
 
         $activity['activity_date_time'] = CRM_Utils_Date::customFormat($values['activity_date_time']);
-        $activity['status_id'] = $activityStatus[$values['status_id']];
+        $activity['status_id'] = CRM_Core_Pseudoconstant::getLabel('CRM_Activity_BAO_Activity', 'status_id', $values['status_id']);
```

Comment:
These aren't the same - it now includes disabled. I personally like that but some people won't.